### PR TITLE
Remove unnecessary TODOs

### PR DESCRIPTION
**Pull request by kanes115**
This PR removes unnecessary TODOs from code

### CODE REVIEW

File: apps/ejabberd/src/gen_mod.erl [hunk 1]
```diff
@@ -229,7 +229,6 @@ wait_for_stop1(MonitorReference) ->
 get_opt(Opt, Opts) ->
     case lists:keysearch(Opt, 1, Opts) of
         false ->
-            % TODO: replace with more appropriate function
```
Comment (kanes115) [inline apps/ejabberd/src/gen_mod.erl]: @ludwikbukowski, any comment on that?

Comment (ludwikbukowski) [reply apps/ejabberd/src/gen_mod.erl]: throwing exception here if fine for me.
Hence "TODO" can be removed

File: apps/ejabberd/src/ejabberd_auth_external.erl [hunk 2]
```diff
@@ -366,7 +366,6 @@ is_fresh_enough(TimeStampLast, CacheTime) ->
 
 %% @doc Code copied from mod_configure.erl
 %% Code copied from web/ejabberd_web_admin.erl
-%% TODO: Update time format to XEP-0202: Entity Time
```
Comment (arcusfelis) [inline apps/ejabberd/src/ejabberd_auth_external.erl]: so, why not to update? :D

Comment (michalwski) [reply apps/ejabberd/src/ejabberd_auth_external.erl]: No need to, we don't return the timestamp to the users. It's used only internally so it's more efficient to operate on the raw value.

**Pull request merged**